karaf-features/src/main/resources/features.xml

```diff
@@ -28,5 +28,6 @@
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/${jackson.version}</bundle>
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson.version}</bundle>
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/${jackson.version}</bundle>
+        <bundle dependency="true">mvn:com.fasterxml.jackson.datatype/jackson-datatype-jdk8/${jackson.version}</bundle>
```

Comment:
It looks to me we are hard wiring a jdk8 dependency into the code and this sounds scary to me :)

Reply:
It's not.  It's an additional module to add support for Jdk 8 data types to Jackson 2, which are  included with Jackson 3.  This is because we use jdk8 Optional types in the alarm data that Jackson 2.x doesn't natively understand.